test/test_fsdp.sh

```diff
@@ -20,16 +20,19 @@ launch() {
     # the NCCL_NET setting is to work around transient issues on a
     #   specific host (`devgpu001.nha2`)
     NCCL_DEBUG=WARN CUDA_VISIBLE_DEVICES=0,1 NCCL_NET=SOCKET python test/test_fsdp.py \
-        --mode fsdp --is_fp8 $IS_FP8
+        --mode fsdp --is_fp8 $IS_FP8 --compile_fsdp $COMPILE --fullgraph $FULLGRAPH
 
     # compare the outputs and state dicts and verify equivalence
-    python test/test_fsdp.py --mode analyze --is_fp8 $IS_FP8
+    python test/test_fsdp.py --mode analyze --is_fp8 $IS_FP8 --compile_fsdp $COMPILE --fullgraph $FULLGRAPH
     echo "Success: ✅"
 
     echo "✅ All Tests Passed ✅"
 }
 
-for IS_FP8 in False True
+# IS_FP8, COMPILE, FULLGRAPH
```

Comment:
is fullgraph always false in this PR? If yes, maybe delete for now and we can add in future PR when we have code for it?

Reply:
yes, because fullgraph compile always fails in compile fsdp test. I kept this option so that we can simply change the options in test_fsdp.sh to test fullgraph again in the future.

Reply:
makes sense, maybe add a comment on why this is here but not used yet